src/main/resources/META-INF/jqassistant-rules/ejb/ejb.xml

```diff
@@ -15,76 +15,57 @@
     </concept>
 
     <concept id="ejb:StatelessSessionBean">
-        <providesConcept refId="ejb:EJB" />
-        <description>Labels all types annotated with @javax.ejb.Stateless with "EJB" and "Stateless".</description>
+        <description>Abstract concept that returns all Stateless Session Beans.</description>
         <cypher><![CDATA[
-            MATCH (t:Type)-[:ANNOTATED_BY]->()-[:OF_TYPE]->(a:Type)
-            WHERE a.fqn="javax.ejb.Stateless"
-            SET t:JEE:EJB:Stateless
+            MATCH (t:Type:JEE:EJB:Stateless)
```

Comment:
Wir sollten uns mal auf einen Einrückungsstil einigen. Wir haben in dem Plugin nun:

MATCH
  ...
WHERE 
  ..
SET 
  ..
RETURN 
 ..

und 
MATCH ..
WHERE ..
SET ..
RETURN ..

Ich persönlich mag ja:
MATCH   ..
WHERE   ..
SET         ..
RETURN ..

Das liest sich für mich am besten und ist kompakt. Wäre aber auch mit der 2. Variante fein, dasa entspricht auch den Styling guidelines: https://neo4j.com/docs/cypher-manual/current/styleguide/

Reply:
Habe ich als allgemeinen Hinweis mit aufgenommen, sehe es aber nicht im Scope dieses PR, daher schließe ich den Thread.